lib/storage/bucket.js

```diff
@@ -198,6 +198,10 @@ Bucket.prototype.delete = function(callback) {
  * var file = bucket.file('my-existing-file.png');
  */
 Bucket.prototype.file = function(name) {
+  if (!name) {
```

Comment:
If we are executing Bucket#file then we for sure have a Bucket object. And when you create a Bucket object, you must specify a name so I think this is not necessary if we are moving the check into Bucket#file.